composer.json

```diff
@@ -21,7 +24,7 @@
     "optimize-autoloader": true
   },
   "scripts": {
-    "check-cs": "@php ./vendor/bin/phpcs --extensions=php --ignore=/vendor/",
-    "fix-cs": "@php ./vendor/bin/phpcbf --extensions=php --ignore=/vendor/"
+    "check-cs": "@php ./vendor/bin/phpcs",
+    "fix-cs": "@php ./vendor/bin/phpcbf"
```

Comment:
Why we no longer `--ignore=/vendor/` by default?

Reply:
Denis told me that this is implemented in the sniffer

Reply:
No since this is defined in the standards 😉

Reply:
@dingo-d you mean you can tell phpcs to use specific settings for sniffing when using a specific standard?

Reply:
yes that is what phpcs.xml.dist is used

Reply:
The `node_modules` `vendor` and some are disabled by default in the Infinum's standards, but you can also add (and should for every project) project specific `phpcs.xml.dist` file where you define the standard you use (in our case our own) plus some project specific overrides 🙂